sw/device/tests/sram_ctrl_sleep_sram_ret_contents_test.c

```diff
@@ -10,43 +10,70 @@
 #include "sw/device/lib/testing/aon_timer_testutils.h"
 #include "sw/device/lib/testing/pwrmgr_testutils.h"
 #include "sw/device/lib/testing/rstmgr_testutils.h"
+#include "sw/device/lib/testing/sram_ctrl_testutils.h"
 #include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 #include "sw/device/silicon_creator/lib/drivers/retention_sram.h"
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
-#define NUM_TEST_WORDS 16
+enum {
+  /**
+   * Retention SRAM start address (inclusive).
+   */
+  kRetSramBaseAddr = TOP_EARLGREY_SRAM_CTRL_RET_AON_RAM_BASE_ADDR,
+
+  // See `sw/device/silicon_creator/lib/drivers/retention_sram.h`.
```

Comment:
Is there a way to get a symbolic constant for that 2048

Reply:
Is there a reason for not using `&retention_sram_get()->reserved_owner`?

If we just want to _publish_ constants, we can move those to chip.h and use them here and in `retention_sram.h`.

Reply:
Is it ok to use silicon owner functions in the top level tests?

Reply:
Yeah I think that is ok. We use the `silicon_creator/lib/drivers/retention_sram.h` functions in other tests as well.

Reply:
Fixed